content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: influxctl
+list_title: influxctl
+description: >
+  The `influxctl` command line interface (CLI) performs administrative tasks in
+  an InfluxDB Cloud Dedicated cluster.
+menu:
+  influxdb_cloud_dedicated:
+    name: influxctl
+    parent: CLIs
+weight: 101
+influxdb/cloud-dedicated/tags: [cli]
+---
+
+The `influxctl` command line interface (CLI) performs administrative tasks in
+an InfluxDB Cloud Dedicated cluster.
+
+- [Usage](#usage)
+- [Commands](#commands)
+- [Flags](#command-flags)
+- [Download and install influxctl](#download-and-install-influxctl)
+- [Configure connection profiles](#configure-connection-profiles)
+- [Authentication](#authentication)
+
+## Usage
+
+```sh
+influxctl [flags] [command]
+```
+
+## Commands
+
+| Command                                                                 | Description                            |
+| :---------------------------------------------------------------------- | :------------------------------------- |
+| [database](/influxdb/cloud-dedicated/reference/cli/influxctl/database/) | Manage InfluxDB databases              |
+| [token](/influxdb/cloud-dedicated/reference/cli/influxctl/token/)       | Manage InfluxDB database tokens        |
+| [version](/influxdb/cloud-dedicated/reference/cli/influxctl/version/)   | Output the current `influxctl` version |
+| [help](/influxdb/cloud-dedicated/reference/cli/influxctl/help/)         | Output `influxctl` help information    |
+
+## Flags {#command-flags}
+
+| Flag        | Description                                                |
+| :---------- | :--------------------------------------------------------- |
+| `--profile` | Specify a connection profile to use (default is `default`) |
+
+---
+
+## Download and install influxctl
```

Comment:
fwiw we are only providing a binary package download, so the instructions should be nearly the same (e.g. download this package, extract it, use the binary).